docs/source/admin/traffic_ops/configuration.rst

```diff
@@ -158,6 +158,9 @@ Many of the settings for the different servers in a Traffic Control CDN are cont
 +--------------------------+---------------+---------------------------------------------------------------------------------------------------------------------------------------+
 | geolocation6.polling.url | CRConfig.json | The location to get the IPv6 GeoLiteCity database from.                                                                               |
 +--------------------------+---------------+---------------------------------------------------------------------------------------------------------------------------------------+
+| maxmind.default.override | CRConfig.json | The destination geo coordinates to use for client location when maxmind returns a default location that matches the country code.     |
```

Comment:
Can we enter more than 1 country at a time? i.e. US and Canada?

Reply:
Yes, this parameter can be specified multiple times with different values. The documentation should be updated to reflect that.

@rivasj can you update the documentation to reflect this?

Reply:
I've updated documentation to show support for multiple parameter values.